changelogs/fragments/66357-support-changing-fetch_url-settings-for-rundeck-modules.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- rundeck modules - support changing fetch_url parameters
```

How about mentioning the new options?
```suggestion
- rundeck modules - added new options ``client_cert``, ``client_key``, ``force``, ``force_basic_auth``, ``http_agent``, ``url_password``, ``url_username``, ``use_proxy``, ``validate_certs`` to allow changing fetch_url parameters.
```

---

That is indeed clearer, yeah. Added.